integration/service/poiloader/loader.go

```diff
@@ -0,0 +1,26 @@
+package poiloader
```

Comment:
It's more like a functional package for service `oasis`, right? Not a new service.

Reply:
Originally, I thought this package might be used by other package.  But you are right, currently the only user is inside of spatialIndexer.  Will put it there not under /service